Firestore/core/src/firebase/firestore/remote/grpc_connection.mm

```diff
@@ -52,6 +54,8 @@
 
 }  // namespace
 
+std::string GrpcConnection::test_certificate_path_;
```

Comment:
I'm not particularly concerned about order of destruction here, especially considering that this is test-only, but feel free to push back. Also, will be happy to discuss alternatives.